src/main/java/com/hubspot/imap/protocol/message/ImapMessage.java

```diff
@@ -26,8 +31,15 @@
   Set<GMailLabel> getGMailLabels() throws UnfetchedFieldException;
   Message getBody() throws UnfetchedFieldException;
 
+  default String bodyToString() throws UnfetchedFieldException, IOException {
+    MessageWriter writer = new DefaultMessageWriter();
```

Comment:
can we not make this static? also I don't love having this in a default method impl. Could we instead have a utility class intended to write out message bodies?